addons/wowl/static/tests/helpers/mock_server.ts

```diff
@@ -19,6 +21,45 @@ import { DBRecord, ORMCommand } from "../../src/services/model";
 // Types
 // -----------------------------------------------------------------------------
 
+/*
+ *  MODELS AND FIELDS DEFINITION
+ */
+
+export type FieldType =
```

Comment:
we will need this in the prod code at some point (but we may move it when this happens)